packages/k4generatorsconfig/package.py

```diff
@@ -23,6 +23,8 @@ class K4generatorsconfig(CMakePackage):
     depends_on("hepmc3")
     depends_on("heppdt")
     depends_on("pythia8")
+    depends_on("python@3.7:")
```

Comment:
```suggestion
    depends_on("python")
```
We don't need to specify not to use a terribly old python, probably it's not possible to build it with 3.7 because of other packages. 3.7 is also deprecated in spack.